playbooks/roles/deploy-acid/templates/supervisor_acid.conf

```diff
@@ -0,0 +1,30 @@
+[program:acid]
+environment=
+    FLASK_ENV={{ flask_env }},
+    TESTING=0,
+{% if dbhost is defined %}
```

Comment:
I don't like this if-ology.
If we really need this variables just put them without IF-statements.
In wors case scenario it will empty string..
Beter idea is to create a variable in ansible

```yaml
ENVS:
  - FLASK_ENV=development
  - TESTING=0
  - DBHOST=127.0.0.1
  - DBUSER=root
  - DBPASS=root
```
And use loop to create list of environments.